templates/nextflow-aurora-mysql.yaml

```diff
@@ -291,7 +310,7 @@ Resources:
       Description: Bootstrap newly Created Aurora Database
       Handler: index.lambda_handler
       Role: !GetAtt BootstrapLambdaRole.Arn
-      Runtime: python3.7
+      Runtime: python3.10
```

Comment:
python3.7 is EOL, it won't even deploy anymore.  Need to update to never version. https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html